### ISSUE TITLE
Build and install contrib classes

### ISSUE THREAD

**Issue by johannesring** [conda-forge/vmtk-feedstock#17]
Hello @ramtingh,

I would like to have contrib classes enabled for vmtk. Contrib scripts are currently enabled, but contrib classes are turned off, which makes scripts like `vmtkdistancetospheres` fail. For example see the following output:
```
$ vmtkdistancetospheres -ifile input.vtp -ofile output.vtp

Automatic piping vmtkdistancetospheres
Parsing options vmtkdistancetospheres
    SurfaceInputFileName = input.vtp
    SurfaceOutputFileName = output.vtp
Explicit piping vmtkdistancetospheres
Input vmtkdistancetospheres members:
    Id = 0
    Disabled = 0
    Surface = None
    SurfaceInputFileName = input.vtp
    DistanceToSpheresArrayName = DistanceToSpheres
    DistanceOffset = 0.0
    DistanceScale = 1.0
    MinDistance = 0.0
    MaxDistance = -1.0
    Opacity = 1.0
    vmtkRenderer = None
    SurfaceOutputFileName = output.vtp
Reading VTK XML surface file.
Executing vmtkdistancetospheres ...
Please position the mouse and press space to add spheres, 'u' to undo
Quit renderer
Traceback (most recent call last):
  File "/home/johannr/miniconda3/envs/vmtk/lib/python3.10/site-packages/vmtk/pypeserver.py", line 48, in RunPypeProcess
    pipe.Execute()
  File "/home/johannr/miniconda3/envs/vmtk/lib/python3.10/site-packages/vmtk/pype.py", line 324, in Execute
    scriptObject.Execute()
  File "/home/johannr/miniconda3/envs/vmtk/lib/python3.10/site-packages/vmtk/vmtkdistancetospheres.py", line 355, in Execute
    self.Surface = self.ComputeDistances()
  File "/home/johannr/miniconda3/envs/vmtk/lib/python3.10/site-packages/vmtk/vmtkdistancetospheres.py", line 91, in ComputeDistances
    distanceToSpheresFilter = vtkvmtk.vtkvmtkPolyDataDistanceToSpheres()
AttributeError: module 'vmtk.vtkvmtk' has no attribute 'vtkvmtkPolyDataDistanceToSpheres'
```
In this example I open a file `input.vtp` using `vmtkdistancetospheres` and a window pops up as expected. I then add some spheres using `space` and then quit the program using `q`. The program fails because `vtkvmtkPolyDataDistanceToSpheres` is not available. This is because contrib classes were not enabled at build time. Setting `-DVTK_VMTK_CONTRIB:BOOL=ON` when configuring vmtk should fix this issue.

I see that contrib classes are explicitly turned off with `-DVTK_VMTK_CONTRIB:BOOL=OFF` on macOS and Windows, so I am not sure if enabling contrib will make the build fail on these platforms. I will make a pull request that enables contrib classes on Linux, but let me know if I should enable it on macOS and Windows as well.

**Comment by ramtingh** [comment]
@johannesring I believe the contrib classes were disabled as they weren't updated for new VTK and causing build issues, however as they seem to work fine on linux I don't think there's any problem with enabling them on linux for now. Thanks for the pull request!

**Comment by johannesring** [comment]
Fixed in PR #18.
